### DISCUSSION TITLE
How to display one author per line in listing

### DISCUSSION THREAD

**Discussion by netw0rkf10w** [quarto-dev/quarto-cli#5838] [Q&A]
### Description

I'm using Quarto's `default` listing and found that the display of the authors is pretty ugly:
 
<img width="479" alt="Screenshot 2023-06-07 at 01 24 13" src="https://github.com/quarto-dev/quarto-cli/assets/8569515/b5b665a6-0880-4106-bbcc-e1966d4a82c4">

Is it possible to display one author per line and without the comma?

Thank you in advance for your help!

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/journals/authors.html>.

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Quarto Playground"
format: html
author:
  - name: Albert Einstein
    affiliation: Princeton University
  - name: Donald Knuth 
---

````

</td>
<td>
<img width="722" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/c3780aed-8288-4e68-8db5-8bb53249b409">
</td>
</tr>
</table>
